src/main/java/Task.java

```diff
@@ -0,0 +1,39 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    protected String symbol;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+        System.out.println("Got it. I've added this task:");
```

Comment:
Maybe it would be better to print the message at a higher level? Like in the Duke class perhaps. It might create issues in the future if a subclass needs to print a different message, or if you are using the class in a different context